### PR TITLE
Ft travelrequest docs

### PR DESCRIPTION
**Pull request by iradukundairenee**
#### What does this PR do?

This PR works on documentation of ft-travel-request
 
#### Description of Task to be completed

create Model for Travel request
Services for controlling the model
route for creating requester trip
route for requester to fetching the trips
#### How should this be manually tested?

1. git clone repo 
2. git checkout ft-travelrequest-Docs
3. run npm i
4. open postgresql locally
5. copy .env.example to .env
6. add all variables as listed in .env
7. npm run start
8. npm run dev 
9. run http://localhost:{PORT}/api-docs/#

#### What are the relevant Trello story?
https://trello.com/c/dVWhW8A7

### CODE REVIEW

**Review by NIRoberto — APPROVED** [review]

**Review by Patricknshimiyimana — APPROVED** [review]

File: src/docs/travelRequest.js [hunk 1]
```diff
@@ -1,13 +1,65 @@
+/**
+* @swagger
+* /api/v1/trip/save:
+*   post:
+*     tags:
+*       - Trips
+*     name: save trip
+*     summary: create trip request
+*     consumes:
+*        - multipart/form-data
```
Comment (nicele08) [inline src/docs/travelRequest.js]: You may use `- application/json`
this link can help you how to de describe Request Body: [swagger docs](https://swagger.io/docs/specification/2-0/describing-request-body/)

File: src/docs/travelRequest.js [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+*         in: header
+*         required: true
+*       - in: formData
+*         name: orgin
+*         type: string
+*         required: true
+*       - in: formData
+*         name: destination
+*         type: Array
+*       - in: formData
```
Comment (nicele08) [inline src/docs/travelRequest.js]: you may pass parameters as body instead of formData

File: src/docs/travelRequest.js [hunk 3]
```diff
@@ -1,13 +1,65 @@
+/**
```
Comment (nicele08) [inline src/docs/travelRequest.js]: Screenshot from 2021-03-22 09-48-11

**Review by nicele08 — CHANGES_REQUESTED** [review]

**Review by mukunzidd — CHANGES_REQUESTED** [review]
- @iradukundairenee why did you delete `.env.example`?
- Please write a good description for the PR

**Review by nicele08 — APPROVED** [review]

**Review by mukunzidd — APPROVED** [review]

**Pull request merged**